CommunityFridgeMapApi/template.yaml

```diff
@@ -216,6 +216,28 @@ Resources:
             RestApiId:
               Ref: ApiGatewayApi
 
+  SendSESFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
```

Comment:
I think we'll need a section for policies

This is an example I saw here: https://medium.com/nerd-for-tech/how-to-use-aws-secret-manager-and-ses-with-aws-sam-a93bb359d45a
```
Policies:
        - SESCrudPolicy: { IdentityName: Sandeep.Yaramchitti }
        - SESCrudPolicy: { IdentityName: ysandeepkumar88@gmail.com }
```

Not sure what the identityName stuff should be for us, need to look into that. I would assume that's out email but wondering if I need to set that up manually

Reply:
Okay I added [info@collectivefocus.site](mailto:info@collectivefocus.site) to our aws account. So you can set SESCrudPolicy to that